src/realm/impl/transact_log.hpp

```diff
@@ -66,15 +66,16 @@ enum Instruction {
     instr_DictionaryInsert = 37,
     instr_DictionarySet = 38,
     instr_DictionaryErase = 39,
+    instr_DictionaryClear = 40,
 
-    instr_SetInsert = 40, // Insert value into set
```

Comment:
Please don't change the existing values.
As the comment on this enum says, any change here is a file format breaking change. So I'd recommend adding the new instruction at the end. You can target the next-major branch and add some migration tests.

Reply:
Thanks for spotting this. It is ok to append it to the end, it is not the right place where it should be, though. Probably we should have left some "space" between the several commands in order to avoid having this breaking changes.

Reply:
> It is ok to append it to the end

Adding a value to the end is still a file format breaking change. Other clients need to know how to interpret this history instruction.

Reply:
Right, good point. These values are not used for encoding sync history, just internal history for notifications. Since we will be breaking the format we can shuffle the values and add it to the logical place. 👍